cli/cmd/common/utils_test.go

```diff
@@ -0,0 +1,42 @@
+package common
+
+import (
+	"embed"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+//go:embed fixtures/consul/* fixtures/consul/templates/_helpers.tpl
+var testChart embed.FS
+
+func TestReadChartFiles(t *testing.T) {
+	files, err := ReadChartFiles(testChart, "fixtures/consul")
+	require.NoError(t, err)
+	foundChart := false
```

Comment:
Kind of silly to suggest stylistic changes in tests, but just in case you want to inline this, they should all default to false.

```suggestion
	var foundChart, foundValues,  foundTemplate, foundHelper bool
```

Reply:
They will all default to `false` confirmed: https://play.golang.org/p/K12S2kvhbks